README.md

```diff
@@ -72,20 +72,44 @@ It's also possible to build and use a locally built image. See the
 
 ## Securing the Docker ACAP using TLS
 
-The Docker ACAP can be run either unsecured or in TLS mode. The Docker ACAP uses
-TLS as default. Use the "Use TLS" dropdown in the web interface to switch
-between the two different modes. It's also possible to toggle this option by
-calling the parameter management API in [VAPIX](https://www.axis.com/vapix-library/) and setting the
-`root.dockerdwrapper.UseTLS` parameter to `yes` or `no`. The following commands would enable TLS:
+The Docker Compose ACAP application can be run in either TLS mode or unsecured mode. The Docker Compose
+ACAP application uses unsecured mode by default. These modes can be used with or without TCP and IPC
```

Comment:
We should still have the secure option as default: 
- for docker-acap Use TLS: "yes" and TCP Socket "yes"
- for docker-compse-acap Use TLS: "yes" and TCP Socket "no"